### ISSUE TITLE
serviceNodePort not found in template

### ISSUE THREAD

**Issue by QuentinN42** [bitnami/charts#10675]
### Name and Version

bitnami/rabbitmq 10.1.5

### What steps will reproduce the bug?

```bash
$ helm install rabbitmq bitnami/rabbitmq
NAME: rabbitmq
...
CHART VERSION: 10.1.5
APP VERSION: 3.10.5** Please be patient while the chart is being deployed **
...
RabbitMQ can be accessed within the cluster on port [HERE] at rabbitmq.saas.svc.cluster.local

To access for outside the cluster, perform the following steps:

To Access the RabbitMQ AMQP port:

    echo "URL : amqp://127.0.0.1:5672/"
    kubectl port-forward --namespace saas svc/rabbitmq 5672:5672
...
```

### What is the expected behavior?

There is only two spaces in the `[HERE]` position...

### What do you see instead?

I think we just need to pass the variable `serviceNodePort`

https://github.com/bitnami/charts/blob/6a85f6ca79f160b3269d24661ff20085be907858/bitnami/rabbitmq/templates/NOTES.txt#L42

To be `servicePort` same as here :

https://github.com/bitnami/charts/blob/6a85f6ca79f160b3269d24661ff20085be907858/bitnami/rabbitmq/templates/NOTES.txt#L130

**Comment by javsalgar** [comment]
Hi!

Thanks for the input! Would you like to submit a PR improving the NOTES.txt?

**Comment by javsalgar** [comment]
Awesome! Thank you so much!

**Comment by QuentinN42** [comment]
Resolved by #10688 .
:tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada:
